package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "to-start-case",
+  "version": "1.0.0",
+  "description": "Convert a string to Start Case for human display purposes.",
+  "main": "dist/to-start-case.js",
```

Comment:
This "main" field is not enough. I need you to understand what the other fields "module", "browser", "esnext" do and what they are for.